server_admin/topics/clients/client-policies.adoc

```diff
@@ -0,0 +1,159 @@
+* Enforce secure signature algorithm for <<_client-credentials,Signed JWT client authentication (private-key-jwt)>> is used
+* Enforce HTTPS redirect URI and make sure that configured redirect URI does not contain wildcards
+* Enforce OIDC `request` object satisfying high security level
+* Enforce Response Type of OIDC Hybrid Flow including ID Token used as _detached signature_ as described in the FAPI 1 specification, which means that ID Token returned from Authorization response won't contain user profile data
+* Enforce more secure `state` and `nonce` parameters treatment for preventing CSRF
+* Enforce more secure signature algorithm when client registration
+
+[[_client_policy_profile]]
+===== Profile
+
+A profile consists of several executors which can realize a security profile like FAPI. Profile can be configured by the Admin REST API (admin console) together with it's executors.
+There are two _global profiles_ configured in the {project_name} by default with pre-configured executors compliant with the FAPI Baseline and FAPI Advanced specifications.
+More details in the FAPI section of the link:{adapterguide_link}#_fapi-support[{adapterguide_name}].
+
+[[_client_policy_policy]]
+===== Policy
+
+A policy consists of several conditions and profiles. The policy can be adopted to clients satisfying any condition of this policy. The policy refers several profiles and all
+executors of these profiles execute their task against the client that this policy is adopted.
+
+
+==== Configuration
+
+Policies, profiles, conditions, executors can be configured by Admin REST API, which means also an admin console. To do so, there is a tab _Realm_ -> _Realm Settings_ -> _Client Policies_
+, which means an administrator can have client policies per realm.
+
+The _Global Client Profiles_ are automatically available in each realm. However there are not client policies
+configured by default. This means that administrator is always required to create any client policy if he wants for example the clients of his realm to be FAPI compliant. Global
+profiles cannot be updated, but administrator can easily use them as a template and create his own profile if he wants to do some slight changes in the global profile configurations.
+There is JSON Editor available in the admin console, which simplifies the creation of new profile based on some global profile.
+
+==== Backward Compatibility
+
+Client Policies can take over Client Registration Policies described in the link:{adapterguide_link}#_client_registration_policies[{adapterguide_name}].
+However, Client Registration Policies also still co-exist. This means that for example during Dynamic Client Registration request to create/update client, both client policies and
+client registration policies are applied.
+
+In the future, Client Registration Policies feature will be removed and the existing client registration policies will be migrated into new client policies automatically.
```

Comment:
Is it definitie?
How about "The current plans are for the Client Registration Policies feature to be removed and ..."

Reply:
I replaced with the formulation you proposed